src/DegenGambit.sol

```diff
@@ -605,6 +605,21 @@ contract DegenGambit is ERC20, ReentrancyGuard {
         }
     }
 
+    // Payout Estimate function to easily display current payouts estimate at time of function call
+    function payoutEstimate(
+    ) external view returns (uint256[6] memory payouts) {
+        payouts[0] = 0;
```

Comment:
What's the point of this?

Reply:
I figured it gave a proper list of payouts based on the document for degen gambit.
For others using non-official websites this is still an option for payout.
Also this allows for the array indexes to follow the document P1, P2, ... , P5 for a more direct understanding and comparison.

Reply:
We decided to remove the `0` payout.